cypress/integration/docs-smoke-test/content-menu.js

```diff
@@ -0,0 +1,26 @@
+import { URL_DOCS_SMOKE_TEST } from '../../support/urls';
+
+describe('Content menu', () => {
```

Comment:
The component is not called "content menu".  The term isn't used anywhere yet.   I suggest to use the specific name of the component tested - also for the test file name.